jcl/src/jdk.management/share/classes/com/ibm/lang/management/OpenJ9DiagnosticsMXBean.java

```diff
@@ -0,0 +1,144 @@
+ * <br>
+ * <table border="1">
+ * <caption><b>Usage example for the {@link OpenJ9DiagnosticsMXBean}</b></caption>
+ * <tr> <td>
+ * <pre>
+ * {@code
+ *   ...
+ *   try {
+ *      mxbeanName = new ObjectName("com.ibm.lang.management:type=OpenJ9Diagnostics");
+ *   } catch (MalformedObjectNameException e) {
+ *      // Exception Handling
+ *   }
+ *   try {
+ *      MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
+ *      if (true != mbeanServer.isRegistered(mxbeanName)) {
+ *         // OpenJ9DiagnosticsMXBean not registered
+ *      }
+ *      OpenJ9DiagnosticsMXBean diagBean = JMX.newMXBeanProxy(mbeanServer, mxbeanName, OpenJ9DiagnosticsMXBean.class);
+ *   } catch (Exception e) {
+ *      // Exception Handling
+ *   }
+ * }
+ * </pre></td></tr>
+ * </table>
+ */
+
+public interface OpenJ9DiagnosticsMXBean extends PlatformManagedObject {
+	/**
+	 * Reset the JVM dump options to the settings specified when the JVM was started removing any additional
+	 * configuration done since then. This method may throw a DumpConfigurationUnavailableException if the dump 
+	 * configuration cannot be altered. If this occurs it will usually be because a dump event is currently being 
+	 * handled.
+	 *
+	 * @throws DumpConfigurationUnavailableException - if the configuration cannot be changed because a dump is already in progress.
+	 */
+	public void resetDumpOption() throws DumpConfigurationUnavailableException;
+
+	/**
+	 * This function sets options for the dump subsystem.
+	 * The dump option is passed in as a String. Use the same syntax as the -Xdump command-line option, with the 
+	 * initial -Xdump: omitted. See Using the -Xdump option as described in the section on dump agents in the 
+	 * documentation for the IBM JVM. This method may throw a DumpConfigurationUnavailableException if the dump
+	 * configuration cannot be altered.
+	 *
+	 * @param dumpOptions - the options string to be set
+	 * @throws InvalidDumpOptionException - if the specified option cannot be set or is incorrect.
+	 * @throws DumpConfigurationUnavailableException - if the configuration cannot be changed becuase a dump is already in progress.
+	 * @throws java.lang.SecurityException - if there is a security manager and it doesn't allow the checks required to change the dump settings
+	 * @throws java.lang.NullPointerException - if options is null
+	 */
+	public void setDumpOptions(String dumpOptions) throws InvalidDumpOptionException, DumpConfigurationUnavailableException;
+
+	/**
+	 * This function triggers the specified dump agent
+	 *
+	 * @param dumpAgent - the dump agent to be triggered
+	 * @throws IllegalArgumentException - if the specified dump agent is invalid or unsupported by this method.
+	 * @throws java.lang.RuntimeException - if the vm does not contain RAS dump support
+	 * @throws java.lang.SecurityException - if there is a security manager and it doesn't allow the checks required to take this dump
```

Comment:
please use different wording, `perform` instead of `take` would work.

Reply:
or `trigger`

Reply:
@pshipton I have taken this description from Dump.java, Should I change take to trigger in that file too?

Reply:
yes, please change it there too